### PR TITLE
OCLOMRS-372: Remove the show all button on add CEIL concepts page.

### PR DESCRIPTION
**Pull request by dessHub**
# JIRA TICKET NAME:
[OCLOMRS-372: Remove the show all button on the add CEIL concepts interface.](https://issues.openmrs.org/browse/OCLOMRS-372)

# Summary:
- Removed "**show all concepts**" button and implement `fetchConcepts` request on `componentDidMount`.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -75,7 +76,9 @@ export class Navbar extends Component {
                   </DropdownMenu>
                 </UncontrolledDropdown>
               </Nav>
-            </Collapse>)}
+            </Collapse>
+          )
+            }
```
Comment (ozezzy) [inline src/components/Navbar.js]: the changes in this file are not needed for this ticket.

File: src/components/dictionaryConcepts/components/ActionButtons.jsx [hunk 2]
```diff
@@ -23,9 +22,6 @@ const ActionButtons = ({
           >
           Edit
           </Link>
-          <AddMapping
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/ActionButtons.jsx]: Is this related related to remove the "show all button"?

Comment (dessHub) [reply src/components/dictionaryConcepts/components/ActionButtons.jsx]: No. I can't remember making these changes. This must be a mistake.

Comment (dkayiwa) [reply src/components/dictionaryConcepts/components/ActionButtons.jsx]: Can you go through all these changes and make sure you do not include any unneeded ones?

Comment (dessHub) [reply src/components/dictionaryConcepts/components/ActionButtons.jsx]: I have taken care of it.

File: src/tests/bulkConcepts/containers/BulkConceptsPage.test.js [hunk 3]
```diff
@@ -225,18 +225,14 @@ describe('Test suite for BulkConceptsPage component', () => {
     expect(mapStateToProps(initialState).preview).toEqual([]);
     expect(mapStateToProps(initialState).loading).toEqual(false);
   });
-  it('should show all concepts', () => {
+  it('should render with at least one concept', () => {
```
Comment (dkayiwa) [inline src/tests/bulkConcepts/containers/BulkConceptsPage.test.js]: Where does this one concept come from?

Comment (dessHub) [reply src/tests/bulkConcepts/containers/BulkConceptsPage.test.js]: Mock file containing mock concepts.

Comment (dkayiwa) [reply src/tests/bulkConcepts/containers/BulkConceptsPage.test.js]: I thought the tested behaviour is for the screen not to load any concepts until when one types a filter.

Comment (dessHub) [reply src/tests/bulkConcepts/containers/BulkConceptsPage.test.js]: Sorry about that this was from the previous implementation when I was fetching concepts on componentDidMount. I have removed it.

Comment (dkayiwa) [reply src/tests/bulkConcepts/containers/BulkConceptsPage.test.js]: So why does it reflect as part of a commit for this ticket?

Comment (dessHub) [reply src/tests/bulkConcepts/containers/BulkConceptsPage.test.js]: It is not supposed to be there.
